pyspecdata/fourier/convolve.py

```diff
@@ -1,12 +1,12 @@
 from ..general_functions import inside_sphinx
-from pylab import r_,fft,ifft,ifftshift,fftshift,exp,ones_like
+from pylab import r_,fft,ifft,ifftshift,fftshift,exp,ones_like,sqrt,pi
 from pyspecdata import init_logging,strm
 logger = init_logging("debug")
 
-def convolve(self,axisname,filterwidth,convfunc = (lambda x,y: exp(-(x**2)/(2.0*(y**2))))):
-    r'''Perform a convolution on the data in its
-    current domain via multiplication in its
-    conjugate domain.
+def convolve(self,axisname,filterwidth,convfunc = (lambda x,y:
+    exp(-(x**2)/(2.0*(y**2)))/(y*sqrt(2*pi)))
```

Comment:
I am not sure where this `(y*sqrt(2*pi))` comes from, otherwise this and the rest of the actual code of the function I agree with

Reply:
that makes the gaussian normalized

Reply:
(so that its integral is normalized)